packages/react/src/breadcrumb/Breadcrumbs.tsx

```diff
@@ -0,0 +1,39 @@
+import { forwardRef } from 'react';
+import { cx } from 'cva';
+import {
+  Breadcrumbs as RACBreadcrumbs,
+  Breadcrumb as RACBreadcrumb,
+  Link as RACLink,
+  type BreadcrumbsProps as RACBreadcrumbsProps,
+  type BreadcrumbProps as RACBreadcrumbProps,
+} from 'react-aria-components';
+import { ChevronRight } from '@obosbbl/grunnmuren-icons-react';
+
+const defaultClasses = 'flex flex-wrap';
+
+type BreadcrumbsProps = {
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+  links: Array<{ href: string; label: string }>;
+} & Omit<RACBreadcrumbsProps<RACBreadcrumbProps>, 'className' | 'style'>;
+
+function Breadcrumbs(props: BreadcrumbsProps) {
+  const { className, links, ...restProps } = props;
+
+  return (
+    <RACBreadcrumbs {...restProps} className={cx(className, defaultClasses)}>
+      {links.map((link, i) => (
+        <RACBreadcrumb key={i} className="flex items-center">
+          <RACLink href={link.href}>{link.label}</RACLink>
+          <ChevronRight className="-mt-0.5 px-1" />
```

Comment:
hmm.. Hvorfor må ikonet dras opp en pixel?

Reply:
Nå får siste crumb rendret en chevron etter seg. Vi kan bruke `[data-current]` for å targete crumb for å skjule ikonet her... Siste crumb skal heller ikke være underlined ser jeg i Figma.

Reply:
Ja, den är inte färdig hehe 😄 

Angående pixeln så var det bara noe jag inte har fjernat än 😅